.github/workflows/scan-pull-request.yml

```diff
@@ -9,7 +9,6 @@ on:
         required: false
 
 permissions:
-  pull-requests: write
   contents: read
```

Comment:
Are these permissions needed at all? I would think the default tokens and such give this read permissions. Maybe not though... can't remember.

Reply:
I think probably not, and it's just leftover from the workflow "template" I copied over, so I'll remove and add it back if necessary.